### PR TITLE
Remove Packet CSI

### PR DESCRIPTION
**Pull request by johananl**
**How to categorize this PR?**

/area storage
/kind cleanup
/priority normal

**What this PR does / why we need it**:

The Packet CSI components seem to be broken at the moment. Since they are being deprecated, it makes more sense to remove them than to try and fix them. An alternative storage solution for Packet will likely be added in the future.

**Release note**:

```other operator
Packet CSI has been removed since it is being deprecated and doesn't currently work anyway.
```

### CODE REVIEW

**Comment by gardener-robot-ci-2** [comment]
Thank you @johananl for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: charts/gardener-extension-provider-packet/templates/configmap-logging.yaml [hunk 1]
```diff
@@ -21,15 +21,8 @@ data:
...
-        Match               kubernetes.*csi-packet-controller*packet-csi*
-        Key_Name            log
-        Parser              kubeapiserverParser
-        Reserve_Data        True
-
     [FILTER]
         Name                rewrite_tag
         Match               kubernetes.*
-        Rule                $tag ^kubernetes\.var\.log\.containers\.(cloud-controller-manager-.+?_.+?_packet-cloud-controller-manager|csi-packet-controller-.+?_.+?_packet-csi) user-exposed.$TAG true
+        Rule                $tag ^kubernetes\.var\.log\.containers\.(cloud-controller-manager-.+?_.+?_packet-cloud-controller-manager) user-exposed.$TAG true
```
Comment (johananl) [inline charts/gardener-extension-provider-packet/templates/configmap-logging.yaml:27]: I wasn't 100% sure about the correctness of the filter syntax after removing the CSI-related stuff (and am not sure how to test it). I'd appreciate if you folks could verify.

**Review by deitch — APPROVED** [review]
LGTM, but feedback from the core gardener team would be appreciate.

File: charts/gardener-extension-provider-packet/templates/configmap-logging.yaml (hunk 1 above)
Comment (rfranzke) [reply charts/gardener-extension-provider-packet/templates/configmap-logging.yaml:27]: FMPOV this looks good, @Kristian-ZH or @vlvasilev can you please confirm?

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by Kristian-ZH — APPROVED** [review]
/lgtm for the logging part

**Pull request merged**